bin/setup.js

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env node
+
+var path = require('path')
+var fs = require('fs')
+
+// use process.env to access npm config environment variables.
+// - https://docs.npmjs.com/misc/config#environment-variables
+// - https://docs.npmjs.com/misc/config#save
+var saveRequested = process.env.npm_config_save
+
+var pathToAppRoot = path.resolve('..', '..')
+
+// This block only executes if Hoodie is installed with `-S` or `--save` flags.
+if (saveRequested) {
+  var packageJson = require(path.join(pathToAppRoot, 'package.json'))
+
+  packageJson.scripts = packageJson.scripts || {}
+  packageJson.scripts[ 'start' ] = 'hoodie'
+
+  var newPackageJson = JSON.stringify(packageJson, null, 2)
+
+  fs.writeFile(path.join(pathToAppRoot, 'package.json'), newPackageJson,
+    function (error) {
+      if (error) {
+        console.error(error)
+        return
+      }
+      console.log('Start your Hoodie app with "npm start"\n')
+    })
+
+  // Create README.md if one is not found.
+  // - https://nodejs.org/api/fs.html#fs_fs_access_path_mode_callback
+  fs.open(path.join(pathToAppRoot, 'README.md'), 'wx',
+    function (error, fd) {
+      if (error) {
+        if (error.code === 'EEXIST') {
+          // console.error('README.md found')
+          return
+        } else {
+          throw error
```

Comment:
@gr2m is it correct to throw this error here or should it be logged like above with package.json?